routing/js/leaflet-hash.js

```diff
@@ -1,218 +1,220 @@
 (function(window) {
-	var HAS_HASHCHANGE = (function() {
```

Comment:
source formatted this. please have a look at this commit for the non whitespace changes:

https://github.com/valhalla/demos/commit/8fb2659530ca0835c6c5f131698f69f0030072ea